source/MaterialXTest/GenOsl.h

```diff
@@ -56,7 +56,7 @@ class OslShaderGeneratorTester : public GenShaderUtil::ShaderGeneratorTester
         whiteList =
         {
             "ambientocclusion", "arrayappend", "backfacing", "screen", "curveadjust", "displacementshader",
-            "volumeshader", "IM_constant_", "IM_dot_", "IM_geomattrvalue"
+            "volumeshader", "IM_constant_", "IM_dot_", "IM_geomattrvalue",
```

Comment:
minor: why the ","?

Reply:
I had modified this before. Will revert.